src/tools/UploadSmartDetectorPackage.ps1

```diff
@@ -91,6 +91,12 @@ elseif ($Environment -eq "DOGFOOD")
     $resourceGroup = 'SmartDetectorAlertsRuleEngineINT'
     $storageAccountName = 'globalsmartdetectorsint'
 }
+elseif ($Environment -eq "FAIRFAX")
+{
+    $subscriptionId = '7f9906c0-20c2-437a-8f76-7402230c9c98'
+    $resourceGroup = 'SmartAlerts-Fairfax-Global'
+    $storageAccountName = 'globalsmartdetectorsff'
+}
```

Comment:
this is internal implementation details of the rule engine - how about we move this script to the RE repo?

Reply:
You're right, I'll move it.